MIGRATION_v2-to-v3.md

```diff
@@ -258,7 +262,8 @@ getOpenChannel('channel-url')
 Can be imported as `import { ChannelList } from "@sendbird/uikit-react"`
 or `import ChannelList from "@sendbird/uikit-react/ChannelList"`
 
-### Changes - none
+### Changes
+  * queries.channelListQuery.MemberStateFilter -> queries.channelListQuery.MyMemberStateFilter
```

Comment:
Do you mean type here? If you are introducing the property of the channelListQuery, we need to say `myMemberStateFilter`